web/src/pages/student/[id].tsx

```diff
@@ -190,20 +229,20 @@ const Student: React.FC<Props> = ({ self }) => {
                     </StatGroup>
                     {dates.map((date: string) => {
                       return (
-                        <Box mt={5} key={date} w="full" border="1px" borderColor="gray.300" borderRadius="md" boxShadow="lg" p="3" rounded="md" bg="white" mb={4}>
-                          <Text fontSize="22">{new Date(date).toLocaleDateString()}</Text>
+                        <Box mt={5} key={date} w="full" border="1px" borderColor="gray.300" borderRadius="md" boxShadow="lg" pt="3" pb="3" rounded="md" bg="white" mb={4}>
+                          <Text fontSize="22" pl="3">{new Date(date).toLocaleDateString()}</Text>
                           <CardTable columns={columns} data={absences.filter(absence => absence.date === date).sort((a, b) => -a.lessonIndex + b.lessonIndex)}
                             keyFn={(row) => row.original.id}
                             rowFn={(row: Row<any>) => (
                               <Flex w="full" transition="all" transitionDuration="200ms" boxShadow="sm" _hover={{ boxShadow: 'md' }} borderRadius="md" alignItems="center" px={4} py={2}>
                                 <Flex flexDirection="column">
-                                  <Text fontWeight="bold">{row.cells[0].render('Cell')}. Stunde</Text>
-                                  <Text>Eingereicht von <span style={{ fontWeight: 'bold' }}>{row.cells[1].render('Cell')}</span></Text>
+                                  <Text fontSize="sm"fontWeight="bold">{row.cells[0].render('Cell')}. Stunde</Text>
+                                  <Text fontSize="sm"><Icon as={FaUser} mr={2} mb={1} />{row.cells[1].render('Cell')}</Text>
                                 </Flex>
                                 <Spacer />
                                 <Flex flexDirection="column">
-                                  {row.cells[3].value ? (<Tag mb={2} bgColor="blue.400" color="white">Klausur</Tag>) : (<></>)}
-                                  {row.cells[3].value ? (<Tag colorScheme="green" variant="solid">Entschuldigt</Tag>) : (<></>)}
+                                  {row.cells[2].value ? (<Tag fontSize="sm" mb={2} bgColor="blue.400" color="white">Klausur</Tag>) : (<></>)}
+                                  {row.cells[3].value ? (<Tag fontSize="sm" colorScheme="green" variant="solid">Entschuldigt</Tag>) : (<Tag fontSize="sm" mb={2} colorScheme="red">Unentschuldigt</Tag>)}
```

Comment:
tags have a size prop https://chakra-ui.com/docs/data-display/tag#props

might be better than changing the font size

Reply:
changed that